services/prefect/flows/nyt_updater.py

```diff
@@ -0,0 +1,34 @@
+from prefect import Flow
+from prefect.schedules import CronSchedule
+
+from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
+from prefect.utilities.notifications import slack_notifier
+from prefect.engine.state import Failed
+
+
+def init_updater_flow():
+    with Flow(
+        "NYTParquetUpdater",
+        schedule=CronSchedule("30 6 * * *"),
```

Comment:
Can you add a comment something like

> NYT typically publishes their daily data update at 1:10AM EST and so we kick off our scraper at 1:30AM EST.

As an aside, I wonder if we're going to have any issues with getting "desynched" from NYT when daylight savings time ends...

I wonder if we should be thinking about trying to implement something like we had in covid-data-public where we just query their repo every 30min to see if they've pushed data.  That way we aren't reliant on this timing working out.

Reply:
Yeah, I agree that it would be ideal to constantly be checking the NYT data

Reply:
(I'll look to add this during my revisions)